### ISSUE TITLE
Does the project support multi-speaker fine-tuning?

### ISSUE THREAD

**Issue by Pushkraj19** [daswer123/xtts-finetune-webui#103]
Hi, thanks for the great work!

I’m wondering if this project supports fine-tuning using datasets that include multiple speakers. For example, if I have a dataset with audio samples from 10 different female speakers and 10 different male speakers, can I fine-tune the model to handle all of them? Or is it designed to work with only one speaker per fine-tuning session?

Would I need to label each speaker separately or provide speaker embeddings?

Thanks in advance!
